.ci/.jenkins_tav.yml

```diff
@@ -3,6 +3,7 @@ TAV:
   - '@hapi/hapi'
   - '@koa/router'
   - apollo-server-express
+  - aws-sdk
```

Comment:
oops we missed this. Ideally the Jenkinsfile would read which tav modules to run from .tav.yml, but I didn't want to get into that right now.